src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js

```diff
@@ -4,42 +4,49 @@ import { withStyles } from "@material-ui/styles";
 import ListView from "./ListView";
 import StyledCard from "./StyledCard";
 
+import CardImg from "../../../../../assets/images/dummy-card.png";
+
 import { useStyles } from "./styles";
 
 class CardGroup extends Component {
     render() {
         const { classes } = this.props;
         return (
             <div className={classes.cardCollection}>
-                <StyledCard
+                <ListView
+                    cardMedia={CardImg}
```

Comment:
In ***future***, while creating the repeated contents of same type.. e.g. ListView is repeating here..  Make an array of objects and map it.

Reply:
i have just created that for a temp purpose, cause the data is going to be coming from the api